src/client/components/CloseButton/CloseButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Cross } from '../icons/Cross'
+
+const Button = styled.button`
+  width: 1.5rem;
+  height: 1.5rem;
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+  background: ${colorsV3.white};
+
+  :focus {
+    outline: none;
+  }
+
+  :hover path {
+    fill: ${colorsV3.gray700};
+  }
+
+  path {
+    fill: ${colorsV3.gray900};
+  }
+`
+
+type Props = React.HTMLProps<HTMLButtonElement>
```

Comment:
Haha I won you over with this one ;) 

Like I said in the other thread - I'm happy with either solution as long as we move to a design system in the near future.

Reply:
Hahah yes Ellen and I came up with a solution that would make both you @robinandeer and @styrelius happy ;)

Reply:
Oh right, since you throw away all the other props 😅 very diplomatic